test/extensions/wasm/test_data/bad_signature_cpp.cc

```diff
@@ -1,20 +1,13 @@
 // NOLINT(namespace-envoy)
 #include <string>
 
-#include "proxy_wasm_intrinsics.h"
+#define EMSCRIPTEN_KEEPALIVE __attribute__((used)) __attribute__((visibility("default")))
```

Comment:
We could probably change it to `PROXY_WASM_KEEPALIVE`, since this won't be exclusive to Emscripten in the long run.

Reply:
Done